validator/project_validator_test.go

```diff
@@ -4394,6 +4394,7 @@ func TestValidateContainers(t *testing.T) {
 			}
 
 			ref := &model.ProjectRef{
+				Id:         "proj",
```

Comment:
GH doesn't let me comment on unchanged lines, but should L4381 also clear the project vars collection?

Reply:
Did this + modified this test to assert that the image expansion works correctly